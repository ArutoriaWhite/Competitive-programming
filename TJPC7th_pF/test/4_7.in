732182 1000
80 0
210 0
325 0
1002 1
1073 1
2302 1
3648 1
4752 0
4908 1
4938 0
5437 0
5532 0
5562 1
6611 0
7127 1
8338 1
8952 0
10045 1
10599 0
10616 1
11049 1
11121 1
11702 0
12084 0
12608 0
13180 1
13455 1
13992 0
14164 0
14168 0
14490 1
15087 1
15676 0
16651 0
17511 1
19064 0
20760 1
22696 1
24469 1
25245 1
25454 0
25988 1
27822 1
28306 0
28524 1
30752 0
32027 1
32379 1
32446 0
36972 1
39849 1
40958 0
41377 1
43330 0
43497 0
43583 1
44694 0
45070 0
46702 0
47050 0
47369 0
47701 0
48423 0
48700 0
50830 0
51636 0
51824 0
53386 1
53578 0
53641 1
53721 1
54420 1
55959 1
57674 1
58080 0
61568 1
61730 1
62584 0
63188 0
64051 0
65433 0
65495 1
65969 0
66090 0
66863 0
66977 0
67693 1
67808 0
68085 0
68163 0
68394 0
68830 0
69490 0
69737 1
69945 1
70455 1
70497 0
70694 0
71837 1
72552 0
74164 0
74588 0
74840 0
77020 0
79330 1
79759 1
79811 0
79943 1
80070 0
80713 0
82268 0
82762 1
83016 1
83905 1
84030 1
85031 1
85435 1
87114 0
87185 1
88742 0
91565 0
92224 1
92237 0
92763 0
93377 1
94293 1
94694 1
95693 1
95894 0
96204 0
96387 1
97264 1
99160 0
99173 0
99518 0
100470 0
101723 1
102711 1
102720 1
103025 0
104420 1
104485 1
105570 0
105602 1
105793 1
106139 0
107300 1
108077 0
108672 1
108716 1
109617 0
111918 0
112374 0
112532 1
112899 1
114569 1
115106 1
116279 0
116965 0
118080 1
118374 0
118501 1
120041 0
120721 1
121178 0
122259 0
125872 1
126053 1
127358 1
127388 1
128132 0
128526 1
129909 1
130175 0
131727 1
133665 0
134963 1
135201 0
135860 0
136301 1
136455 0
137867 0
139902 0
140422 0
140574 1
141296 1
142084 1
142458 0
143408 0
145229 0
145304 1
145416 0
145677 0
147911 0
148225 1
149019 1
149391 0
149757 0
150406 0
151678 0
152057 0
152269 1
153330 0
153439 0
153493 0
153911 0
154659 1
156157 0
156613 0
156677 0
157704 1
158831 1
159001 1
159230 1
159511 1
159664 1
159784 1
162742 0
163448 0
165316 1
165737 0
166018 0
167408 0
168073 1
168845 1
169655 0
170898 1
171730 0
171986 1
172494 1
172647 1
173371 1
173582 1
174441 1
174897 0
175024 1
175815 0
175924 0
175971 1
176472 1
176525 0
177072 1
178175 0
178394 1
178727 0
179241 0
180217 1
180765 0
180930 1
181130 1
181632 0
182708 1
184277 1
185239 0
187687 1
188159 0
188242 1
189742 0
189812 0
192080 1
192483 0
194487 0
195182 1
195804 0
196478 1
196770 1
197116 0
197189 1
198016 1
200489 0
201657 0
202152 0
202230 1
202442 0
203349 0
203481 1
205359 1
205611 0
206263 0
206500 1
206585 0
206626 0
206792 1
207734 1
208504 1
209782 1
210329 0
211089 1
211674 1
211798 0
213001 1
213120 0
215031 1
215289 1
215431 1
215459 1
216702 1
217096 1
217629 0
218252 0
220423 1
220514 0
220747 1
221603 0
222075 1
223283 1
224507 1
225026 0
226559 0
226769 0
227880 0
227974 0
228372 0
228579 1
229159 0
231160 0
231583 0
232021 1
234594 1
234598 1
235923 0
236071 0
237436 0
237651 1
237946 1
238804 1
239107 1
239230 1
240065 0
241234 0
241595 0
243164 0
243286 0
243731 1
243769 1
244039 1
244665 1
244960 1
246321 0
247168 0
247573 1
249071 1
250502 0
252152 0
252358 0
253220 0
253314 1
253388 1
253890 1
255046 0
255090 1
256040 0
258078 0
258515 0
258579 0
259093 1
259112 1
259208 1
259215 0
260446 0
262196 1
262934 0
263327 0
263830 1
264578 0
265015 1
266767 1
266808 0
267616 0
269117 0
272050 0
272079 1
272326 1
272537 0
272557 0
273087 1
273551 0
275621 0
276557 0
276699 0
277416 0
277709 1
279300 1
280028 1
280320 1
280729 0
281245 0
281834 1
282502 0
283449 0
283486 0
284005 1
285194 0
285357 1
285979 1
287374 0
288617 1
288693 0
289567 0
290627 0
291402 0
291678 1
292373 1
293487 1
294735 1
295975 1
296014 0
297276 1
297574 1
298885 0
300587 1
301078 0
301300 0
302208 0
302579 0
302585 1
303109 0
303888 1
305236 0
305307 1
305611 0
306143 0
307702 0
309345 1
309813 0
310144 1
310470 0
311556 0
311813 1
312384 1
312857 0
312869 1
313732 0
315739 1
318026 1
318046 0
321608 0
325625 1
325819 1
327284 1
330118 0
330185 1
330672 0
331728 1
331860 1
332024 1
333480 1
333926 0
334107 1
334923 1
336181 0
336844 1
337882 0
338406 0
339046 0
339415 1
339429 1
339586 0
340259 1
341066 0
341730 0
342132 1
342947 1
343720 0
343914 1
345873 1
346292 1
346319 0
346342 1
346472 0
346935 0
347259 1
348138 0
348649 1
348998 0
349751 1
350538 1
350976 0
352503 1
354303 0
354353 1
357195 1
358186 0
358206 1
359658 1
360077 0
360632 1
360842 1
363965 1
364955 0
365306 1
365718 0
365921 0
366502 0
367565 1
368001 0
368159 1
368582 1
369500 1
370873 0
370917 1
372676 1
375556 0
375602 1
376342 0
376469 0
377436 1
377555 0
378137 1
378459 1
379984 0
380040 0
380814 0
381037 1
381110 0
381111 0
381561 0
381579 1
382844 1
382868 1
383005 1
383193 0
384707 1
386147 0
386589 0
386612 0
387239 0
387600 1
387821 0
388290 0
388741 0
389523 1
389991 0
390190 1
390905 1
392859 0
393111 0
394112 0
394985 1
396015 1
396366 0
396683 0
396799 1
397110 1
397441 1
398640 1
399354 1
399616 1
400823 1
402417 0
404343 1
404640 1
405243 0
407300 0
408596 0
409472 1
409591 0
409695 0
412404 0
413138 1
413924 1
414036 1
417458 0
420059 1
420359 0
421416 1
421737 1
422296 1
422637 1
422845 0
424383 0
424695 0
424751 1
425653 1
425701 1
425749 0
425874 1
426467 1
427344 1
428047 1
429167 1
430896 0
431291 0
431497 1
432039 0
432289 1
433331 1
433590 1
433682 0
434214 1
434355 1
436461 1
436466 0
437640 0
438011 1
442632 0
443214 1
443615 0
443878 1
444303 1
444998 0
445020 0
445178 1
445352 1
448645 0
448859 0
452671 0
452918 0
453180 0
453964 1
454800 1
456122 1
456613 1
459362 1
460142 0
460198 1
462208 1
462674 0
463187 0
463289 0
466292 1
468034 1
468870 1
470426 0
471868 1
472533 0
472762 1
473148 1
473708 0
474469 0
475275 0
475742 1
476863 1
476993 1
478349 1
479154 0
481008 1
481660 1
482482 0
482798 0
484572 1
485104 1
485630 0
485741 1
485751 1
486166 0
486517 1
486813 0
486848 0
487352 0
489880 1
490735 0
491535 1
491726 0
493916 1
494101 1
494302 0
494402 1
494924 1
497305 0
500082 1
500501 1
500952 1
500981 1
501286 0
502035 0
502131 1
502764 0
502941 1
502994 0
506105 0
506568 1
506886 0
507056 0
507672 1
507826 1
508750 1
509624 0
511597 0
513746 0
514221 0
515876 0
518982 0
520685 1
522237 1
522482 1
523238 1
523667 1
523779 0
524470 1
524472 1
524519 1
524642 1
525108 1
525660 1
526071 1
526120 1
526682 0
526898 1
528122 1
528201 1
529041 1
529263 1
529929 1
530285 1
530729 1
531897 1
532572 1
534086 0
534649 0
539056 0
541988 1
543294 1
543626 1
545580 1
545787 0
545992 1
546327 1
546954 1
547498 1
547569 0
548028 0
548291 0
548775 0
550031 1
550689 0
551898 1
553064 0
553463 1
555643 1
555958 0
556299 0
556856 0
558119 0
559706 1
561761 1
561765 1
562522 0
562776 0
562891 1
562915 1
564199 1
564348 0
564943 0
566028 0
567260 0
567532 1
567970 1
569138 0
569339 1
572177 0
572264 0
573592 1
573919 0
574224 1
574337 0
574648 0
574930 0
574938 0
574957 0
575509 0
575614 0
575714 0
575771 1
575981 1
576387 0
576932 0
576987 1
578099 1
578800 1
579122 0
579510 1
579760 1
579800 1
579978 0
579985 1
582023 0
582532 0
583353 1
583883 1
583948 1
584141 0
584671 1
584674 0
584872 0
585093 1
585333 1
586146 1
586548 0
586782 0
589010 1
589090 0
592147 1
592227 1
592426 0
592512 0
593125 1
593641 1
593726 1
594414 0
594573 0
595313 1
595530 1
597132 1
598027 0
598253 0
598878 1
599498 1
599718 0
599998 1
600940 1
601347 0
602023 1
602326 0
602529 0
603655 0
604019 1
604211 1
604596 1
606095 1
606216 1
607434 1
607552 1
607668 0
607888 0
607984 1
608630 0
608862 1
610355 0
610777 0
610812 1
611087 1
611179 1
612598 1
613210 0
614907 0
615142 0
615499 0
616834 0
616838 0
617898 1
618646 1
619135 0
620063 0
624690 1
624692 0
624854 0
625638 1
626738 0
626943 1
626958 1
627316 1
629551 1
630828 0
630842 0
630919 0
631942 1
633182 1
634523 0
634753 0
634994 0
635367 1
635466 1
635853 0
637209 1
638971 0
639150 1
641484 1
641963 0
643240 0
644647 1
645269 0
645554 1
647292 1
647905 0
648083 1
648326 0
649646 1
649656 1
649762 0
650068 0
650477 0
650960 0
651253 0
651387 0
651646 0
651839 1
651958 0
653036 1
653084 1
653345 0
655050 1
656047 0
657046 0
657300 1
658949 0
658996 0
659002 0
659141 1
659592 1
660809 0
660811 1
661286 1
661433 0
662471 0
662549 0
663416 1
663579 0
664547 0
665519 1
666704 1
667035 1
668023 1
668054 1
668179 0
668473 0
668481 0
669361 0
669515 0
669985 1
670019 0
670786 1
672446 1
674877 0
675105 1
675937 0
677445 0
677460 1
678981 0
679851 1
680081 0
682219 1
682670 0
684349 1
684620 1
684994 1
685347 0
685387 0
685524 0
685961 0
686867 1
687170 1
687359 0
688058 0
688127 0
689300 0
689812 1
690724 1
691993 1
693168 1
694531 1
696289 0
696505 1
696739 1
697258 1
697635 1
697697 0
698328 1
699663 1
700353 1
701188 1
701678 0
701694 1
702212 1
702375 1
703736 1
703858 1
704221 0
705489 0
705560 1
706416 1
706448 1
707165 1
707401 1
707471 0
708712 1
711003 0
712437 1
712516 1
712631 0
713073 1
713186 1
713591 0
715139 0
715420 0
715503 0
716483 1
717610 1
718056 1
719042 1
719307 1
721272 1
724032 0
725366 1
727461 1
727655 1
728634 0
730972 1
731951 0
731986 1
