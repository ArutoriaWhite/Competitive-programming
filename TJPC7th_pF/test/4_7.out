453180 725220
